### PR TITLE
feat

### PR DESCRIPTION
**Pull request by ckahub**
#homepage

### CODE REVIEW

File: pages/homepage.html [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+                <button class="btn"><a href="#">
+                    Explore
+                </button>
+            </div>
+              
+            <!-----featured image end-->
+        </header>
+
+    <!--latest collection start--->
+        <section class="latest">
```
Comment (junaidshaikh-js) [inline pages/homepage.html:70]: You can use `main` element to wrap main content of your webpage

**Review by junaidshaikh-js — COMMENTED** [review]
I would suggest hosting your image somewhere and use the link in your webpage.

File: pages/homepage.html (hunk 1 above)
Comment (ckahub) [reply pages/homepage.html:70]: ya but section is too fine for now
i will do image link thing

File: pages/homepage.html [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+            <div class="newProduct"></div>
+            <h1>Latest Collection</h1>
+            <p>Design are essence to your clothes.With new clothes ,you carry new feeling ,new emotion, new decision.Present yourself in variety of clothes</p>
+        
+    <!--latest collection end----->
+
+    <!---new card start-->
+    <div class="card_Display">
```
Comment (ravikumar0403) [inline pages/homepage.html:78]: Keep class names consistent

File: pages/productpage.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://meta-library.netlify.app/main.css">
+    <link rel="stylesheet" href="../style/productpage.css">
+    <title>Document</title>
```
Comment (ravikumar0403) [inline pages/productpage.html:10]: Update title to be relevant with the page

File: style/homepage.css [hunk 4]
```diff
@@ -0,0 +1,184 @@
+:root{
+    --primary-color:black;
+    --secondary-color:tan;
+    --active-color:aqua;
+}
+body{
+    background-color: tan;
```
Comment (ravikumar0403) [inline style/homepage.css:7]: use variable for this color

File: style/homepage.css [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+    padding: 0%;
+    border: 0%;
+    margin: auto;
+}
+
+
+.navigation ul a{
```
Comment (ravikumar0403) [inline style/homepage.css:14]: try to avoid styling html tags directly

File: style/homepage.css [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+    height: 1rem;
+    line-height: 1rem;
+    justify-content:space-evenly;
+    align-items:center;
+    box-shadow: none;
+    text-align: center;
+}
+.nav2 ul{
+    list-style: none;
+    /* text-align: center; */
```
Comment (ravikumar0403) [inline style/homepage.css:35]: remove uncommented code if not required

File: style/homepage.css [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+    background-color: tan;
+    color: var(--primary-color);
+    font-weight: bold;
+    font-family: Arial, Helvetica, sans-serif;
+    width: 14rem;
+    height: 2rem;
+}
+.add2cart:hover{
+    font-size: large;
+    color: #055252;
```
Comment (ravikumar0403) [inline style/homepage.css:145]: use css variable for this color as well

File: style/homepage.css [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+.wishlist:hover{
+    font-size: large;
+}
+.card_Container{
+    justify-content: space-between;
+    margin: 0rem 1rem;
+    margin-bottom: 1rem;
+}
+footer{
+    background-color: #172b4df3;
```
Comment (ravikumar0403) [inline style/homepage.css:156]: here as well

**Review by ravikumar0403 — APPROVED** [review]
I've added a few sanity comments you can check out

File: style/homepage.css (hunk 6 above)
Comment (ckahub) [reply style/homepage.css:35]: ok i done

**Pull request merged**